study1/week1/third-maximum-number.java

```diff
@@ -0,0 +1,28 @@
+// https://leetcode.com/problems/third-maximum-number/
+class Solution {
+    // Time Complexity: O(n), n: length of nums/ Runtime: 3ms
+    // Space Compexity: O(n), n: length of nums/ Memory: 42.47MB
```

Comment:
When we're calculating Space complexity, we don't take into account the size of the array which was already given. So this is `O(1)` as you have several variables.